utils/docker_nlt.sh

```diff
@@ -10,4 +10,7 @@ set -e
 
 ./utils/setup_daos_admin.sh
 
+# Needed for Ubuntu
+export LD_LIBRARY_PATH=/opt/daos/prereq/release/spdk/lib/
```

Comment:
@jolivier23 This looks to be a build/scons issue

Reply:
Hmm, ok.   Someone not adding a proper dependence probably.